charts/trusted-certificate-issuer/templates/role.yaml

```diff
@@ -81,14 +81,7 @@ rules:
   - list
   - update
   - watch
-- apiGroups:
```

Comment:
@poussa  The `secrets/finalizers` are needed for TCI to set/unset finalizers on the secrets it creates. So, this change is not required.

Reply:
@poussa Please ignore my above comment. The required 'patch' rule on secrets was missing which gets fixed by this change.